Makefile

```diff
@@ -31,12 +36,16 @@ verify-dependency-bounds-sys:
 	${SET_LOWER_BOUNDS} ${MANIFEST_PATH_ARGS}
 	RUSTFLAGS="${DENY}" cargo build ${MANIFEST_PATH_ARGS} --all-features
 
+test-compare-fedora-versions:
+	echo "Testing that COMPARE_FEDORA_VERSIONS environment variable is set to a valid path"
+	test -e "${COMPARE_FEDORA_VERSIONS}"
+
 check-fedora-versions:
-	${COMPARE_FEDORA_VERSIONS} ${MANIFEST_PATH_ARGS} \
+	${COMPARE_FEDORA_VERSIONS} ${MANIFEST_PATH_ARGS} ${FEDORA_RELEASE_ARGS} \
```

Comment:
Make the two check-fedora-versions targets depend on the test-compare-fedora-versions target and this is ready to go.

Reply:
OK, I see it now.  It was hidden among the other details in the diff.  I'll make sure I add this in the other PRs.